extra/completions/_alacritty

```diff
@@ -11,6 +11,7 @@ _arguments \
   "--print-events[print all events to stdout]" \
   '(-v)'{-q,-qq}"[reduce the level of verbosity (min is -qq)]" \
   "--ref-test[generate ref test]" \
+  "--hold[Remain open after child process exits]" \
```

Comment:
`Remain` -> `remain`, for consistency with the rest of zsh complitions.

Reply:
Good catch, fixed